src/app/containers/LiveRadioPage/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { string, shape } from 'prop-types';
+import { Headline } from '@bbc/psammead-headings';
+
+import { Grid, GridItemConstrainedMedium } from '../../lib/styledGrid';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+const LiveRadioPage = props => {
+  const { service, match } = props;
+  const { brandName } = useContext(ServiceContext);
+
+  return (
+    <Grid>
```

Comment:
```suggestion
  <main role="main">
      <Grid>
```

I'd suggest wrapping the grid in `<main>` with the aria role set for accessibility purposes. Also we could follow the naming conventions that frontpage and articles have with `Article + ArticleMain` and `FrontPage -> FrontPageMain` therefore renaming this from `LiveRadioPage` to `MediaPageMain`. 

If in the future we need seperate logic per page type we can refactor at that point.